packages/single-invoice/src/utils/getConversionPaymentValues.ts

```
@@ -0,0 +1,168 @@
+ * @param amount Number to convert, may float
+ * @param token Token
+ */
+export const bigAmountify = (
+  amount: number,
+  token: Pick<CurrencyTypes.CurrencyDefinition, "decimals">,
+): BigNumber => {
+  let [whole, decimals] = amount.toString().split(".");
+  let pow = "0";
+  let powSign = true;
+
+  if (decimals && decimals.includes("e")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)